smarts/core/agent_manager.py

```diff
@@ -303,7 +304,7 @@ def send_observations_to_social_agents(self, observations):
             obs = observations[agent_id]
             self._remote_social_agents_action[agent_id] = remote_agent.act(obs)
 
-    def switch_initial_agent(self, agent_interface):
+    def switch_initial_agents(self, agent_interface):
         self._initial_interfaces = agent_interface
```

Comment:
The params need to also be updated I think.
```suggestion
    def switch_initial_agents(self, agent_interfaces):
        self._initial_interfaces = agent_interfaces
```